exosphere-shared/src/docker-helper.ls

```diff
@@ -48,6 +48,38 @@ class DockerHelper
       done null, map((.RepoTags |> head |> (.split ':') |> head), images)
 
 
+  @get-dangling-images = (done) ->
+    docker.list-images {"filters": '{"dangling": ["true"]}'}, (err, images) ->
+      | err => done err
+      done null, images
+
+
+  @force-remove-images = (images, done) ->
+    if images.length == 0 then done! else
+      num-removed = 0
+      for image in images
+        docker.get-image(image.Id).remove {force:true}, (err) ->
+          | err => done err
+          num-removed += 1
+          if num-removed == images.length then done!
+
+
+  @get-dangling-volumes = (done) ->
+    docker.list-volumes (err, volumes) ->
```

Comment:
You have to first run an image that mounts a volume, such as: 
```
docker run -v {some_path}:/data/db  mongo
```
Stop and remove that container, then you should get a dangling volume.